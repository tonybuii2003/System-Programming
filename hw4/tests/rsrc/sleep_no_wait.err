[00000.000000] STARTUP
[00000.000034] PROMPT
[00000.100257] INPUT run echo hello world
[00000.100444] CHANGE 366622: none -> running
[00000.100558] PROMPT
[00000.101175] SIGNAL 17
[00000.101204] CHANGE 366622: running -> stopped
[00000.200817] INPUT run sleep 1
[00000.201032] CHANGE 366623: none -> running
[00000.201117] PROMPT
[00000.201691] SIGNAL 17
[00000.201719] CHANGE 366623: running -> stopped
[00000.301360] INPUT cont 1
[00000.301421] CHANGE 366623: stopped -> running
[00000.301440] PROMPT
[00000.401587] INPUT cont 0
[00000.401632] CHANGE 366622: stopped -> running
[00000.401658] PROMPT
hello world
[00000.402838] SIGNAL 17
[00000.402875] CHANGE 366622: running -> dead
[00000.501924] INPUT quit
[00000.501987] CHANGE 366623: running -> killed
[00000.502389] SIGNAL 17
[00000.502431] CHANGE 366623: killed -> dead
[00000.502447] SHUTDOWN
