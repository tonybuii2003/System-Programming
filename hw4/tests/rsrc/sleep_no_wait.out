0	366622	T	running		echo hello world
0	366622	T	stopped		echo hello world
1	366623	T	running		sleep 1
1	366623	T	stopped		sleep 1
1	366623	T	running		sleep 1
0	366622	T	running		echo hello world
0	366622	T	dead	0x0	echo hello world
